.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Feature request
+about: Suggest an idea for this library
+
+---
+
+*Please consider making a Pull Request if you are capable of doing so.*
+
+**Description what you'd like to happen:**
+
+A clear description if the feature or behavior you'd like implemented.
+
+**Example code of espresso**
```

Comment:
This sounds like mandatory to me, and not all Barista users should know about espresso... maybe you should ask for "example espresso code if available" or something like that

Reply:
Done